f600233d1b154d507b782f84604bcb12/update/sys_metadata_link_32343365974b11106fa8b4b3f153afde.xml

```diff
@@ -6,29 +6,30 @@
     <x_g_dis_atat_service_offering action="INSERT_OR_UPDATE">
         <description/>
         <name>Application</name>
+        <offering_type>XAAS_SERVICE</offering_type>
```

Comment:
Kind of related to the XaaS services so I put it on this line. The `dow_task_number` was added in https://github.com/dod-ccpo/atat-snow/pull/307 for tables like `Training Estimate`, should a `dow_task_number` column also be added to the `Environment Instance` table to be extended to tables like `Compute Environment Instance` or was that left out intentionally to be handled a different way in the UI?

Reply:
I believe that for environment instances, that information will be stored in `IGCE Estimate.dow_task_number` which pre-existed #307, but it does serve a similar purpose to the columns added in that PR.